sorc/ncep_post.fd/GPVS.f

```diff
@@ -65,40 +53,28 @@ SUBROUTINE GPVS
 !-----------------------------------------------------------------------
                            FUNCTION FPVS(T)
 !-----------------------------------------------------------------------
-!$$$  SUBPROGRAM DOCUMENTATION BLOCK
-!                .      .    .
-! SUBPROGRAM:    FPVS        COMPUTE SATURATION VAPOR PRESSURE
-!   AUTHOR: N PHILLIPS            W/NP2      DATE: 30 DEC 82
-!
-! ABSTRACT: COMPUTE SATURATION VAPOR PRESSURE FROM THE TEMPERATURE.
-!   A LINEAR INTERPOLATION IS DONE BETWEEN VALUES IN A LOOKUP TABLE
-!   COMPUTED IN GPVS. SEE DOCUMENTATION FOR FPVSX FOR DETAILS.
-!   INPUT VALUES OUTSIDE TABLE RANGE ARE RESET TO TABLE EXTREMA.
-!   THE INTERPOLATION ACCURACY IS ALMOST 6 DECIMAL PLACES.
-!   ON THE CRAY, FPVS IS ABOUT 4 TIMES FASTER THAN EXACT CALCULATION.
-!   THIS FUNCTION SHOULD BE EXPANDED INLINE IN THE CALLING ROUTINE.
-!
-! PROGRAM HISTORY LOG:
-!   91-05-07  IREDELL             MADE INTO INLINABLE FUNCTION
-!   94-12-30  IREDELL             EXPAND TABLE
-!   96-02-19  HONG                ICE EFFECT
-!
-! USAGE:   PVS=FPVS(T)
-!
-!   INPUT ARGUMENT LIST:
-!     T        - REAL TEMPERATURE IN KELVIN
-!
-!   OUTPUT ARGUMENT LIST:
-!     FPVS     - REAL SATURATION VAPOR PRESSURE IN KILOPASCALS (CB)
-!
-! COMMON BLOCKS:
-!   COMPVS   - SCALING PARAMETERS AND TABLE COMPUTED IN GPVS.
-!
-! ATTRIBUTES:
-!   LANGUAGE: FORTRAN 90
-!   MACHINE:  IBM SP
-!
-!$$$
+!> fpvs() computes saturation vapor pressure.
+!>
+!> Compute saturation vapor pressure from the temperature.
+!> A linear interpolation is done between values in a lookup table
+!> computed in GPVS. See documentation for FPVSX for details.
+!> Input values outside table range are reset to table extrema.
+!> The interpolation accuracy is almost 6 decimal places.
+!> On the CRAY, FPVS is about 4 times faster than exact calculation.
+!> This function should be expanded inline in the calling routine.
+!>
+!> @param[in] T real temperature in Kelvin.
+!> @param[out] FPVS real saturation vapor pressure in kilopascals (CB).
```

Comment:
What is CB?

Reply:
@WenMeng-NOAA @HuiyaChuang-NOAA 
Do you know what are the (CB) stand for?
line 67: FPVS real saturation vapor pressure in kilopascals (CB).
line 139: FPVSX real saturation vapor pressure in kilopascals (CB).

Reply:
@kayeekayee I would defer this question to @HuiyaChuang-NOAA.

Reply:
This kind of confusion is exactly the reason that we should be very cautious using abbreviations in documentation!